lib/macos_ui.dart

```diff
@@ -2,7 +2,8 @@ library macos_ui;
 
```

Comment:
Run `dartfmt` and optimize imports on this file

Reply:
I'll create a pr organizing this file after my other 3 prs get merged